### ISSUE TITLE
Calling Caja with a file path should open the directory and select that file

### ISSUE THREAD

**Issue by CreamyCookie** [mate-desktop/caja#1025]
For example, `caja /home/user/test.txt` should open the home directory and select `test.txt`. (This is what Nautilus does.)

**Comment by cpicon92** [comment]
Agreed. Eventually this could enable browsers to "open containing folder" with the file pre-selected like on other OSs. Possibly also adding a `--select` flag, like nautilus now has would be good as an alternative.
